jsons/Nations.json

```
@@ -703,6 +703,42 @@
+        "startIntroPart2": "Oh sleeping condor, your people call upon you to rise once more and lead them to greatness. Will you answer their plea and unite the diverse lands of Chile under your banner? Will you restore the proud traditions of your ancestors and build an empire that will endure through the ages?",
+        "declaringWar": "I have seen your treachery, and now it is time to act! To arms, my patriots! Let us march to war!",
+        "attacked": "How dare you challenge me! You will regret this day.",
+        "defeated": "You may claim victory today, but know that the spirit of Chile can never be truly vanquished!",
+        "introduction": "I am Bernardo O’ Higgens, the embodiment of Chile's strength and defiance. No one dares cross me without consequence!",
+        "neutralHello": "May the earth bless your steps.",
+        "hateHello": "What do you want now?",
+        "tradeRequest": "Let us join forces and reap the rewards of cooperation.",
+        "outerColor": [25, 25, 112],
+        "innerColor": [189, 183, 107],
+        "uniqueName": "Mapuche Resistance",
+        "uniques": [
+            "Gain control over [All] tiles in a [1]-tile radius <upon building a [Fishing Boats] improvement>",
+            "Gain control over [All] tiles in a [1]-tile radius <upon building a [Dry Dock] improvement>",
+            "Free [Great Admiral] appears <upon discovering [Compass] technology>",
+            "Comment [[Sword] and [Gunpowder] Units  inflict a -25% Combat Penalty to adjacent enemy [Water] Units]",
+            "Comment [[Melee Water] Units inflict this penalty to adjacent enemy [Land] Units]",
+            "[Sword] units gain the [By Reason] promotion <hidden from users>",
+            "[Gunpowder] units gain the [By Reason] promotion <hidden from users>",
+            "[Melee Water] units gain the [By Force] promotion <hidden from users>"
+        ],
+        "cities": [
+            "Santiago de Chile","Valparaíso","Concepción","Antofagasta","Viña del Mar","Temuco","Talca","La Serena","Rancagua","Puerto Montt",
+            "Arica","Quilpué","Calama","Osorno","Talcahuano","Iquique","San Antonio","Los Angeles","Coquimbo","Valdivia",
+            "Chillán","Punta Arenas","Copiapó","Curicó","Temuco","Puerto Varas","Los Andes","Ovalle","Melipilla","San Fernando"
+        ],
+        "spyNames": [
+            "Mateo","Isidora","Diego","Paulina","Sebastián","Catalina","Don Ramón","Juana","Fernando","María Inés"
```

```suggestion
            "Mamo Contreras","Arellano Stark","Paul Schafer","Alvaro Corbalán","Odlanier Mena","Humberto Gordon","Hugo Salas","Gustavo Abarzúa","Ingrid Olderock","La Pepa"
```
Don't ask me why they have full first and last names. I don't know either